plugins/toolbox-search/src/toolbox_search.ts

```diff
@@ -157,19 +157,22 @@ export class ToolboxSearchCategory extends Blockly.ToolboxCategory {
   override setSelected(isSelected: boolean) {
     super.setSelected(isSelected);
     if (isSelected) {
-      this.searchField.focus();
+      this.searchField?.focus();
       this.matchBlocks();
     } else {
-      this.searchField.value = '';
-      this.searchField.blur();
+      const searchField = this.searchField
+      if(searchField){
+        searchField.value = '';
+      }
+      this.searchField?.blur();
     }
   }
```

Comment:
I think in this case it's more readable to return early rather than using `?.` repeatedly.

```ts
super.setSelected(isSelected);
if (!this.searchField) return;
if (isSelected) {
// etc...
```

Reply:
Yes, you're right. Added the check for the object before instead of using ? repeatedly. Let me know if there's anything else @BeksOmega .